test/input/beamville/beam-urbansimv2-static.conf

```diff
@@ -0,0 +1,375 @@
+include "../common/akka.conf"
+include "../common/metrics.conf"
+include "../common/matsim.conf"
+
+beam.agentsim.simulationName = "beamville-urbansimv2_input"
+beam.agentsim.agentSampleSizeAsFractionOfPopulation = 1.0
+beam.agentsim.firstIteration = 0
+beam.agentsim.lastIteration = 0
+
+beam.agentsim.thresholdForWalkingInMeters = 100
+beam.agentsim.thresholdForMakingParkingChoiceInMeters = 100
+beam.agentsim.schedulerParallelismWindow = 30
+beam.agentsim.timeBinSize = 3600
+beam.agentsim.startTime = "00:00:00"
+beam.agentsim.endTime = "30:00:00"
```

Comment:
In order to create a config file that has only few config values different there is no need to copy all the content, you migh just include the base config file in the beginning and then put into config file only the config keys you want to change.
For example - test/input/beamville/beam-csv.conf